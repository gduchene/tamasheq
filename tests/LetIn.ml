let x =
  let y = 1 in

  2 * y
