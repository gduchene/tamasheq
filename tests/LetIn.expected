LetIn_0	=	[1] {
    (no name)	=	2
}
